model/logs/azure.yaml

```diff
@@ -0,0 +1,98 @@
+groups:
+  - id: az.resource.log
+    stability: experimental
+    type: event
+    name: az.log
+    brief: >
+      Describes Azure Resource Log event, see
+      [Azure Resource Log Top-level Schema](https://learn.microsoft.com/azure/azure-monitor/essentials/resource-logs-schema#top-level-common-schema)
+      for more details.
+    attributes:
+      - ref: event.name
+      - ref: az.service_request_id
+      - ref: cloud.resource_id
+    # Future Note: When the build tools support this definition please uncomment and validate the details
+    # included here and what has been added to the manual markdown table
```

Comment:
I didn't find this manual markdown table?

Reply:
Added markdown files in https://github.com/open-telemetry/semantic-conventions/pull/1028/commits/7a9d4fe591d01051dfa34873cca6182eecf68044 which addes the markdown tables 👍🏻